.editorconfig

```diff
@@ -9,4 +9,13 @@ indent_style = space
 indent_size = 4
 
 [*.json]
+end_of_line = lf
```

Comment:
I wouldn't repeat `end_of_line` for each non-`*` case. Rather put `end_of_line` and `insert_final_newline` into `[*]`. If that doesn't fuck up binary files